client/status/status.go

```diff
@@ -26,7 +22,7 @@ type Status any
 // Note: direct assignment may not be compatibility-safe.
 func ErrFromStatus(st Status) error {
```

Comment:
btw, are there any other ways to use that func? should it be exported now? can it be an unexported part of `client` pkg?

Reply:
The function is part of `status` package, not `client` one. I'm not sure it is a good idea right now to move this function and make it unexported

Reply:
BTW, this means that ErrToStatus has to be changed as well. Let's create a separate issue for ErrFromStatus/ErrToStatus/IsSuccessful, this needs to be reviewed. Most likely they're no longer needed at all.

Reply:
#417.